src/plugins/kernel_install/systemd_bootloader.sh

```diff
@@ -0,0 +1,51 @@
+SYSTEMD_BOOT_ENTRIES_DIR="/boot/loader/entries"
+
+function run_bootloader_update()
+{
+  local flag="$1"
+  local target="$2"
+  local name="$3"
+  local sudo_cmd
+  local loader_conf="${SYSTEMD_BOOT_ENTRIES_DIR}/kw_${name}_linux.conf"
```

Comment:
Maybe check if the `name` variable exists before you concatenate it in the loader_conf.